### PR TITLE
Fix TestDiracCommands in GangaLHCb nightlies

### PR DESCRIPTION
**Pull request by mesmith75**
Was missing the credential requirements.

As an aside, does anyone know the purpose of this - it looks to be duplicating the test in the GangaDirac.

### CODE REVIEW

**Comment by alexanderrichards** [comment]
Can one of the admins verify this patch?

**Comment by mesmith75** [comment]
retest this please

**Comment by alexanderrichards** [comment]
Can one of the admins verify this patch?

File: python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def load_config():
 
 @pytest.yield_fixture(scope='class')
 def dirac_job(load_config):
+    from GangaDirac.Lib.Credentials.DiracProxy import DiracProxy
```
Comment (alexanderrichards) [inline python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py]: Is there a good reason to have this import here? If so fine but in general it violates PEP8

File: python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py [hunk 2]
```diff
@@ -94,8 +95,11 @@ def dirac_job(load_config):
     """
     api_script = dedent(api_script)
 
+    cred_req = DiracProxy(group='lhcb_user')
```
Comment (rob-c) [inline python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py]: Erm, you may have to copy https://github.com/ganga-devs/ganga/pull/917/files#diff-b063f6a99352c1604f128062072de4f9R161 wrt  `encodeDefaultProxyFileName=False` option as the default config is _not_ consistent with the lhcb expected use which can make running this manually a pain. I don't know if this comes to bite us here as well or not, you may be saved by the fact you're being explicit in passing the cred along to the Dirac layer.

File: python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py (hunk 1 above)
Comment (mesmith75) [inline python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py]: No - I'll put it at the top

Comment (mesmith75) [inline python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py]: Actually it won't work if that import is at the top - I think Ganga has to have setup the config first.

File: python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py (hunk 2 above)
Comment (mesmith75) [inline python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py]: Do I need to change the user? Which group is the certificate on the Jenkins server a member of?

**Comment by egede** [comment]
This is the output of the `gangage` proxy that is on the Jenkins box.

```
Added VOMS attribute /lhcb/Role=user
Uploading proxy for lhcb_user...
Uploading proxy for private_pilot...
Proxy generated:
subject      : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=gangage/CN=370370/CN=Robot: General Ganga-Project/CN=2166844100/CN=843890156
issuer       : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=gangage/CN=370370/CN=Robot: General Ganga-Project/CN=2166844100
identity     : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=gangage/CN=370370/CN=Robot: General Ganga-Project
timeleft     : 23:53:58
DIRAC group  : lhcb_user
rfc          : True
path         : /tmp/x509up_u12230
username     : gangage
properties   : NormalUser
VOMS         : True
VOMS fqan    : ['/lhcb/Role=user']

Proxies uploaded:
 DN                                                                                            | Group         | Until (GMT)
 /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=gangage/CN=370370/CN=Robot: General Ganga-Project | lhcb_user     | 2018/03/07 14:56
 /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=gangage/CN=370370/CN=Robot: General Ganga-Project | private_pilot | 2018/03/07 14:56
```

File: python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py (hunk 1 above)
Comment (alexanderrichards) [inline python/GangaLHCb/test/Unit/DiracAPI/TestDiracCommands.py]: fine

**Comment by alexanderrichards** [comment]
FYI I don't think that the imperial Dirac instance is enabled for the lhcb vo. with that certificate can you create a gridpp proxy with 
```bash
dirac-proxy-init -g gridpp_user -M
```

**Comment by alexanderrichards** [comment]
> FYI I don't think that the imperial Dirac instance is enabled for the lhcb vo. with that certificate can you create a gridpp proxy with
> 
> dirac-proxy-init -g gridpp_user -M

ok sorry for the noise, failed to spot this was an LHCb test

**Comment by alexanderrichards** [comment]
sorry Mark, do we want the change from `lhcb_user` to `gridpp_user` if this is an lhcb test?

**Comment by mesmith75** [comment]
@alexanderrichards No - I am just messing around trying to figure out why I can't get this to work.

**Comment by alexanderrichards** [comment]
> @alexanderrichards No - I am just messing around trying to figure out why I can't get this to work.

ok np, sorry again for all the confusion I caused :wink:

**Comment by egede** [comment]
This has been fixed elsewhere

**Pull request closed without merging**